lib/rucio/daemons/replicarecoverer/suspicious_replica_recoverer.py

```diff
@@ -416,26 +403,16 @@ def run_once(heartbeat_handler: Any, younger_than: int, nattempts: int, vos: Opt
                                         checksum += 1
                                         files_to_be_declared_bad.append(recoverable_replicas[vo][rse_key][replica_key])
                                         break
-                            else:
-                                if datatype_found_in_file:
-                                    logger(logging.WARNING, "RSE: %s, replica name %s, surl %s: Match for the metadata 'datatype' (%s) of replica found in policy library, but the associated action is invalid. No aciton will be taken.",
-                                           rse_key, replica_key, recoverable_replicas[vo][rse_key][replica_key]['surl'], file_metadata_datatype)
-                                elif scope_found_in_file:
-                                    logger(logging.WARNING, "RSE: %s, replica name %s, surl %s: Match for the specific scope (%s) of replica found in policy library, but but the associated action is invalid. No aciton will be taken.",
-                                           rse_key, replica_key, recoverable_replicas[vo][rse_key][replica_key]['surl'], file_scope)
-                                elif wildcard_found_in_file:
-                                    logger(logging.WARNING, "RSE: %s, replica name %s, surl %s: Wildcard-match for the scope (%s) of replica found in policy library, but but the associated action is invalid. No aciton will be taken.",
-                                           rse_key, replica_key, recoverable_replicas[vo][rse_key][replica_key]['surl'], file_scope)
                         else:
                             # If no policy has been set, default to ignoring the file (no action taken).
                             files_to_be_ignored.append(recoverable_replicas[vo][rse_key][replica_key])
 
             logger(logging.INFO, '(%s) Remaining replicas (pfns) that will be ignored:', rse_key)
             for i in files_to_be_ignored:
-                logger(logging.INFO, 'Ignore: RSE: %s    Scope: %s    Name: %s    Datatype: %s    PFN: %s', rse_key, i["scope"], i["name"], i["datatype"], i["surl"])
+                logger(logging.INFO, 'Ignore: RSE: %s    Scope: %s    Name: %s    Datatype: %s    PFN: %s   kibana_ignore', rse_key, i["scope"], i["name"], i["datatype"], i["surl"])
```

Comment:
I don't really like this `kibana_ignore`. Isn't it possible to just filter on `Ignore` ?